vllm/lora/request.py

```diff
@@ -18,7 +18,7 @@ class LoRARequest:
 
     lora_name: str
     lora_int_id: int
-    lora_local_path: str
+    lora_path: str
```

Comment:
can we keep `lora_local_path` as a deprecated alias for `lora_path`? let's warn people when it's used so they update, but ideally we should avoid API breakage here

Reply:
Just notice this is the user facing api if users use vllm server instead of openai server. Sounds good. I will add deprecation notice there.